src/ngrx/16-test-used-selectors/test-used-selectors.md

```diff
@@ -1,8 +1,35 @@
 @page learn-ngrx/test-used-selectors Testing Selector Use
 @parent learn-ngrx 16
 
-@description Learn how to write unit tests a component making use of NgRx selectors.
+@description Learn how to write unit tests for a component making use of NgRx selectors.
 
 @body
 
-TODO
\ No newline at end of file
+> **Quick Start**: You can checkout [this branch](https://github.com/bitovi/angular-ngrx-chat/tree/use-selectors) to get your codebase ready to work on this part.
+
+## Running Tests
```

Comment:
do we want to repeat this before every test? We might explain it once and then link to the explanation if the user needs it (more maintainable if we want to change this running tests guide)

Reply:
That's true. I think it'll be easy enough to maintain for now. In the next iteration, we'll be redoing these pages when explaining how to write tests. For now, the instructions kinda serve as fluff to make the page more full :^)